enterprise/frontend/src/embedding-sdk/cli/constants/messages.ts

```diff
@@ -65,19 +51,26 @@ export const getMetabaseInstanceSetupCompleteMessage = (instanceUrl: string) =>
   Read more: https://www.metabase.com/docs/latest/installation-and-operation/information-collection
 `;
 
-export const NOT_ENOUGH_TENANCY_COLUMN_ROWS = `
-  At least ${HARDCODED_USERS.length} rows with valid tenancy columns are needed for sandboxing.
-  You can add your tenant's IDs to the "customer_id" user attribute in settings.
-`;
-
-export const getExpressServerGeneratedMessage = (filePath: string) => {
-  const NPM_INSTALL_DEPS_COMMAND = chalk.blue(
-    "npm install express express-session jsonwebtoken cors node-fetch@2",
-  );
+export const getNoTenantMessage = (unsampledTableNames: string[]) => {
+  const tables = unsampledTableNames.join(", ");
+  const warningTitle = `Sandboxing is not configured for the following tables: ${tables}.`;
 
   return `
-  Generated an example Express.js server in "${filePath}".
-  Add the dependencies with "${NPM_INSTALL_DEPS_COMMAND}"
-  Start the server with "node ${filePath}".
+  ${yellow(warningTitle)}
+
+  At least one tenant is needed to demonstrate sandboxing.
+  You can assign your tenant's id to your user attribute, e.g. "customer_id: 5".
 `;
 };
+
+export const SETUP_PRO_LICENSE_MESSAGE = `
+  This tool can set up permissions for multi-tenancy and a mock back-end server that
+  signs users into Metabase to emulate the experience from different tenants.
+
+  To proceed, you will need a Pro license.
+  If you skip, we will proceed without multi-tenancy or SSO.
+`;
+
+export const SDK_LEARN_MORE_MESSAGE = `All done! 🚀 Learn more about the SDK here: ${green(
```

Comment:
This last step message is unaligned 
<img width="1031" alt="image" src="https://github.com/user-attachments/assets/5fe5cec8-7148-43c9-a71b-502c4f578004">

Reply:
Good eye! Let me align it